src/test/ui/issues/issue-49040.stderr

```diff
@@ -1,16 +1,23 @@
-error: expected item, found `;`
+error: statements cannot reside in modules
   --> $DIR/issue-49040.rs:1:28
    |
 LL | #![allow(unused_variables)];
-   |                            ^ help: remove this semicolon
+   |                            ^
+   |
+   = note: the program entry point starts in `fn main() { ... }`, defined in `main.rs`
+   = note: for more on functions and how to structure your program, see https://doc.rust-lang.org/book/ch03-03-how-functions-work.html
+help: consider moving the statements into a function
+   |
+LL | #![allow(unused_variables)]fn my_function() { }
+   |                            ^^^^^^^^^^^^^^^^^^^^
```

Comment:
This is enough of a regression that it should be accounted for, IMO.

Reply:
Would suggest https://github.com/rust-lang/rust/pull/69445#discussion_r383591643 to fix it.